src/libs/Navigation/OnyxTabNavigator.tsx

```diff
@@ -24,11 +25,14 @@ type OnyxTabNavigatorProps = OnyxTabNavigatorOnyxProps &
         /** A function triggered when a tab has been selected */
         onTabSelected?: (newIouType: string) => void;
 
+        tabBar: (props: TabSelectorProps) => React.JSX.Element
+
         screenListeners?: ScreenListeners<NavigationState, MaterialTopTabNavigationEventMap>;
     };
+      
 
 // eslint-disable-next-line rulesdir/no-inline-named-export
-export const TopTab = createMaterialTopTabNavigator();
+export const TopTab:any = createMaterialTopTabNavigator();
```

Comment:
Is this PR WIP?

Reply:
Ohh, Sorry. I think it's left accidently. Let me fix it.